src/s3am/cli.py

```diff
@@ -141,6 +142,9 @@ def add_common_arguments( sp ):
                           "Without this flag, s3am will err on the side of caution and "
                           "exit with an error if it detects unfinished uploads." )
 
+    upload_sp.add_argument( '--overwrite', action='store_true',
+                            help='If the object already exists in S3, should it be overwritten?' )
```

Comment:
"Overwrite the object at DST_URL if it already exists. Without this option, the existing object will not be changed and an the program exits with an error."